### DISCUSSION TITLE
Unable to upload BOM via REST Api

### DISCUSSION THREAD

**Discussion by kartik-579** [DependencyTrack/dependency-track#1554] [Q&A]
Hi,

I am trying to upload BOM via REST API (using Postman) but not able to view the uploaded file on dashboard. Weird thing is I am getting 200 as the response status code. I am using swagger spec as a reference and went through the same for checking any anomalies in my request but unable to find any. 

My request in cURL looks like this - 

```
curl --location --request POST 'http://localhost:8080/v1/bom' \
--header 'X-Api-Key: vFrvAHcs2didewjbcx--------' \
--form 'project="6eb45eac-5fd2-----------------"' \
--form '=@"/Users/test-bom.json"'
```

**Comment by AbdelHajou** [comment]
Your request URL is missing the /api/ part, the URL should be http://localhost:8080/api/v1/bom. You're getting a 200 response because any URL without /api will redirect you to the documentation page.

Are you sure your API is listening on port 8080? The default docker-compose deployment has the front-end on 8080 and the API server on 8081
